gulpfile.babel.js

```diff
@@ -61,11 +80,14 @@ gulp.task('clean-gem', () => {
   return del(`${gemOutput}/**/*.js`);
 });
 
+gulp.task('clean-styleguide', () => {
+  return del(lintConfig);
+});
+
 
 let karmaConfig = {
   configFile: path.join(__dirname, 'karma.conf.js'),
-  singleRun: true,
-  sourceMaps: false
```

Comment:
This should be handled with the `karma-source-maps-loader` plugin now, I'll double check that this is not necessary.